<?xml version="1.0" ?>
<launch>
  <arg name="target"      default="Goesser_27_56_"/>

  <!--Start detection-->
  <node pkg="detection_localisation" type="frame_detection" name="frame_detection_node" output="screen" >  
    <param name="path_package"              type="str"    value="$(find learn_object)/model/" /> 
    <param name="targetName"                type="str"    value="$(arg target)" /> 
    <param name="learning_data"             type="str"    value="$(arg target)12_learning_data.bin" /> 
    <param name="objectName"                type="str"    value="Goesser" /> 
    <param name="verbose"                   type="bool"   value="true" /> 
    <param name="numberImagesPerPose"       type="int"    value="19" />

    <!-- <param name="broadcast_transform"       type="bool"   value="true" />
    <param name="proj_error_threshold"      type="double" value="25" />  -->
  </node>


</launch>
